frontend_react/src/shared/ui/components/custom/small-card.tsx

```diff
@@ -0,0 +1,78 @@
+import { Check, Edit, MoreHorizontal } from 'lucide-react';
+
+export type SmallCardProps = {
+  title: string;
+  limit: string;
+};
+
+export const SmallCard = ({ title, limit }: SmallCardProps) => {
+  const BORDER_LENGTH: number = 25;
+  const truncatedTitle: string =
+    title.length > BORDER_LENGTH
+      ? title.slice(0, BORDER_LENGTH) + '...'
+      : title;
+
+  const edit = () => {
+    console.log('edit');
+  };
+
+  const more = () => {
+    console.log('moreInfo');
+  };
+
+  const done = () => {
+    console.log('done');
+  };
+
+  return (
+    <div className="relative flex h-[155px] w-[179px]">
+      {/* TODO: `bg-[#fbfbfb]` をカラーテーマ使って書き換える */}
+      <div className="small-card-container relative z-0 flex h-[155px] w-[179px] bg-[#fbfbfb]">
+        <div className="z-10 p-3">
+          <p className="pt-1 text-[12px] font-bold text-[#909489]">{limit}</p>
+          <p className="max-w-[155px] pt-6 font-[Noto_Sans_JP] text-[15px] leading-[20px] font-[600]">
+            {truncatedTitle}
+          </p>
+        </div>
+        <button
+          onClick={() => {
+            more();
+          }}
+          className="absolute bottom-[10px] left-[15px] z-10"
+        >
+          <MoreHorizontal size={20} />
+        </button>
+        <button
+          onClick={() => {
+            edit();
+          }}
```

Comment:
eventHandlerを上で定義しているので

```suggestion
          onClick={handleEdit}
```